hack-the-resume/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './styles/App.css';
+import resume from './resume/seed'
+import Basics from './components/Basics'
+import Body from './components/Body'
+
+function App() {
+  let myResume = resume
```

Comment:
Since you are importing the data from your seed file, you technically already have a variable for it so it's redundant to declare this within the function. When passing down the data as props to your component you can just say `<Basics basics={resume.basics}`